src/mocks/api/auth.js

```
@@ -18,4 +18,8 @@ export const postGoogleLogin = async (_idToken) => {
     refresh_token: "mock-refresh-token",
     user_info: MOCK_USER,
   };
+};
+
+export const postLogout = async (_refreshToken) => {
```

ESLint

[error] '_refreshToken' is defined but never used.

(no-unused-vars)